### DISCUSSION TITLE
Any way to completely start over? (Ubuntu 18.04)

### DISCUSSION THREAD

**Discussion by potatomutt** [carla-simulator/carla#4269] [Q&A]
Is there a way to safely, completely, remove carla and the UnrealEngine after successfully make PythonAPI-ing? 
I got that to work only to find out I was working from UE 4_26 instead of 4_24 and now make launch is probably never going to work in this state.

I want to clear up everything and start from scratch, but I don't see a way to revert back to that state.

**Comment by corkyw10** [comment]
You should be able to delete the directory where you built Unreal and then when you want to build CARLA again, run `make clean` before starting the process to remove any intermediate binaries that were created during the previous build.
